API/Entities/AppUser.cs

```diff
@@ -2,12 +2,15 @@ namespace API.Entities
 {
     public class AppUser
     {
+
         public int Id { get; set; }
 
         public string Username { get; set; }
 
         public byte[] PasswordHash { get; set; }
         
         public byte[] PasswordSalt { get; set; }
+
+        public List<Saldo> Saldo { get; set; }
```

Comment:
IEnumerable a nie List, jak się da to staraj się korzystać z interfejsów. Poza tym to jest lista, kolekcja a nazwałeś to liczbą pojedynczą - bardzo mylące